conf/nginx.conf

```diff
@@ -18,6 +29,8 @@ server {
     # When requesting a path without extension, try it, and return the index if not found
     # This allows for client-side route handling
     location / {
-        try_files $uri $uri/ /index.html$is_args$args;
+        # The index file references CSS and JS bundles, so we need to make sure it's always up to date
+        expires -1;
+        try_files $uri $uri/ /index.html;
```

Comment:
I have removed `$is_args$args`, because those actually make no sense when loading a static file. They are meant to be used in fast-cgi contexts, like with PHP and such, where there might be some server-side logic run before nginx returns the result.